docs/index.md

```diff
@@ -1,16 +1,20 @@
 # Overview
-DIDComm is an open protocol for decentralized communication. You can use it to create
-trustworthy relationships between people and businesses in a privacy-preserving way.
+DIDComm is an open protocol for decentralized communication. It uses DIDs ([Decentralized Identifiers](https://www.w3.org/TR/did-core/)) to establish confidential, ongoing connections, without the need for usernames and passwords. You can then layer on existing services and applications on top of DIDComm to take advantage of DIDComm's security and privacy.
 
-It uses DIDs ([Decentralized Identifiers](https://www.w3.org/TR/did-core/)), to establish confidential, ongoing connections, without
-the need for usernames and passwords. You can then layer on existing services and applications
-on top of DIDComm to take advantage of DIDComm's security, privacy and decentralization.
-Read more about DIDComm and its benefits.
+DIDComm empowers end users with privacy-focused solutions by default, while still retaining flexibility and ease of implementation for developers.
+
+## For decision makers
+With DIDComm, you keep your options open. It can work alongside OpenID4VC. It enables features you’re likely to need, such as trustable, ongoing connections—if not now, then in the future.
+- Deliver on privacy promises: connections are established without the need to “phone home”, and the connections are direct and encrypted without involving a middleman.
+- Use DIDComm connections for identity proofing, messaging, verifiable credential exchanges, and more.
+- Choose from many flexible solutions for human-to-human or human-to-business interactions.
+
+## For developers
+DIDComm developer benefits include:
 
-## Developers
 - Open standards protocol, continually improved by an active community.
-- Fully decentralized with no central infrastructure.
-- Transport independent: supports https, websockets, Bluetooth, and more.
+- Fully decentralized with no API.
```

Comment:
I like the overall statement here, but `no API` might not be the clearest way to say it.